Jumpscale/sal/sshd/tests/test_sshd.py

```diff
@@ -0,0 +1,42 @@
+import subprocess 
+from Jumpscale.sal.tls.TLS import TLS
+from Jumpscale import j
+import uuid
+OP_ADD = '+'
+OP_DEL = '-'
+OP_ERS = '--'
+
+def test_add_key():
+    random_key =  str(uuid.uuid4()).replace('-', '')[:10]
+    j.sal.sshd.addKey(random_key)
+    assert random_key not in j.sal.sshd.keys    
+    j.sal.sshd.commit() 
+    assert random_key in j.sal.sshd.keys    
+    authorized_keys = j.sal.sshd.ssh_authorized_keys_path.bytes()
+    assert random_key in authorized_keys
+
+def test_remove_key():
+    random_key =  str(uuid.uuid4()).replace('-', '')[:10]
+    j.sal.sshd.addKey(random_key)
+    j.sal.sshd.commit()
+    j.sal.sshd.deleteKey(random_key)
+    assert random_key in j.sal.sshd.keys    
+    j.sal.sshd.commit()
+    assert random_key not in j.sal.sshd.keys    
+    authorized_keys = j.sal.sshd.ssh_authorized_keys_path.bytes()
+    assert random_key not in authorized_keys
+
+def test_remove_all_keys():
+    random_key1 =  str(uuid.uuid4()).replace('-', '')[:10]
+    random_key2 =  str(uuid.uuid4()).replace('-', '')[:10]
+    j.sal.sshd.addKey(random_key1)
+    j.sal.sshd.addKey(random_key2)
+    j.sal.sshd.commit()
+    assert random_key1 in j.sal.sshd.keys    
+    j.sal.sshd.erase()
```

Comment:
Instead of using j.sal.sshd, can you import the class so it is consistent with other tests, and makes the test more isolated. You can check an example here https://github.com/threefoldtech/jumpscaleX/blob/master/Jumpscale/sal/tls/tests/test_tls.py#L19